spec/ttml1.xml

```diff
@@ -13244,6 +13243,53 @@ shown in <specref ref="fragment-streaming-graphic"/>.</p>
 <p>This specification does not define a transport buffer model or a decoder
 capabilities model.</p>
 </note>
+</div2>
+<div2 id="temporal-fragmentation">
+<head>Temporal fragmentation</head>
+<p>Another possible means by which TTML Content may be streamed is to
+partition a <emph>Document Instance</emph>'s information set into
+temporally bound fragments, each of which is itself a document instance that contains all the
+front matter (head) and content required to present it, where the temporal interval for each
+fragment is constrained, either by the <emph>Document Interchange Context</emph> or by the 
+<emph>Document Processing Context</emph> or by both, such that temporal overlaps with other 
+fragments are resolved.</p>
+<p>For example, some document processing contexts resolve temporal overlap by specifying that a
+maximum of one document instance can be active at any moment in the timeline, and additionally
+specify precedence rules to establish which document instance that is, in case there are multiple
+candidates.</p>
+<p>In order to ensure that all the content is correctly displayed, any content that falls into an
+Intermediate Synchronic Document (ISD) whose interval overlaps with more
```

Comment:
Change ``Intermediate Synchronic Document (ISD)</emph>`` to a link to terminology section (with lowercase).